src/plugins/data/common/search/expressions/kibana_filter.test.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { ExecutionContext } from 'src/plugins/expressions/common';
+import { functionWrapper } from './utils';
+import { kibanaFilterFunction } from './kibana_filter';
+
+describe('interpreter/functions#kibanaFilter', () => {
+  const fn = functionWrapper(kibanaFilterFunction);
+  let context: ExecutionContext;
+
+  beforeEach(() => {
+    context = {
+      getSearchContext: () => ({}),
+      getSearchSessionId: () => undefined,
+      types: {},
+      variables: {},
+      abortSignal: {} as any,
+      inspectorAdapters: {} as any,
+    };
+  });
+
+  it('returns an object with the correct structure', () => {
```

Comment:
Same here, `fn` and `context` are defined outside of the text case, and the test case name just says that something should work "correctly", but it is not clear what.